components/p3a/p3a_config.cc

```diff
@@ -36,15 +37,18 @@ void MaybeReadStringFromCommandLine(base::CommandLine* cmdline,
   }
 }
 
-void MaybeReadURLFromCommandLine(base::CommandLine* cmdline,
-                                 const char* switch_name,
-                                 GURL* config_value) {
+void MaybeReadURLFromCommandLineAndCheck(base::CommandLine* cmdline,
+                                         const char* switch_name,
+                                         GURL* config_value) {
```

Comment:
return absl::optional instead of out param https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs